components/atoms/Price/styles.module.scss

```
@@ -0,0 +1,17 @@
+@import "assets/styles/colors.scss";
```

se puede importar asi, solo una sugerencia a tener en cuenta
```suggestion
@import "assets/styles/colors";
```